tests/factories.py

```diff
@@ -1,11 +1,13 @@
+from datetime import date
+
 import factory
 
 from .models import EncryptedModel
 
 
 class EncryptedModelFactory(factory.DjangoModelFactory):
     """Factory to generate hashed and encrypted data."""
-    class Meta:
+    class Meta:  # NOQA
```

Comment:
`noqa` is not needed in the `tests` module as it's ignored by [coverage](https://github.com/incuna/django-pgcrypto-fields/blob/e3a7d28c3eceb21341dba606ceca771ff3c9e40c/.coveragerc#L3).

This is what coverage report (from travis):
```
Name                     Stmts   Miss  Cover   Missing
------------------------------------------------------
pgcrypto/__init__.py         8      0   100%
pgcrypto/aggregates.py      26      2    92%   59-65
pgcrypto/fields.py          38      0   100%
pgcrypto/forms.py            9      0   100%
pgcrypto/lookups.py         64      4    94%   60-63
pgcrypto/mixins.py          47      0   100%
pgcrypto/models.py           0      0   100%
pgcrypto/proxy.py           20      0   100%
------------------------------------------------------
TOTAL                      212      6    97%
```